projects/plugins/jetpack/modules/search/instant-search/components/path-breadcrumbs.jsx

```diff
@@ -8,10 +8,16 @@ import './path-breadcrumbs.scss';
 function splitDomainPath( path ) {
 	const splits = path.split( '/' ).filter( piece => piece.length > 0 );
 	splits.shift(); // Removes domain name from splits; e.g. 'jetpack.com'
-	return splits.length === 0 ? [ '/' ] : splits;
+	return splits.length === 0 ? [] : splits;
```

Comment:
splits is no worse than an empty array, so I'd just `return splits;`

Reply:
Ah, right you are! [Fixed it up](https://github.com/Automattic/jetpack/pull/20500/commits/ccd1064176f14f291e7ddf1c0a0ca38d70049bb8)